pkg/loader/load.go

```diff
@@ -181,6 +206,10 @@ func (s *loaderImpl) GetSafeMode() bool {
 }
 
 func (s *loaderImpl) markSuccess(txns ...*Txn) {
```

Comment:
does `markSuccess` take effect  synchronously or asynchronously?

Reply:
`markSuccess` can take effect asynchronously, but if we get applied ts here, can only run synchronously after execute DML and DDL